docs/editors-guide/imports.md

```diff
@@ -10,32 +10,18 @@ See [Design Pattern](https://mondo.readthedocs.io/en/latest/editors-guide/e-desi
-4. Run command: `./run.sh make imports/foo_import.obo` 
-    1. For example: `./run.sh make imports/ncbitaxon_import.obo`  
-    1. Note - if you don't have Docker installed but have Robot installed, run this command:  
-  `make imports/foo_import.obo` 
-    1. For example:`make imports/ncbitaxon_import.obo`  
-    1. Note - this can take a while to run.  
-    1. Note - if it failed with a message saying the source ontology is not available, try again later.  
-5. Do the diff on ncbitaxon_import. We expect new classes and some subClassOf changes. A new foo_import.obo file will be created in the imports folder (for example: ncbitaxon_import.obo). Run command: `git diff`  
-6. If you are expecting new classes to be added, check to see if those were added. For example: check the coronovirus shows up in the new import
-    1. `grep 2697049 imports/ncbitaxon_import.obo`
-    2. 2697049 = ID for class I am trying to import.  
-7. Commit the foo_import.* file (for example: `ncbitaxon_import.obo` and `ncbitaxon_import.owl`) (if you see changes on any other files, discard the changes on the other files):  
-     `git add imports/foo_import.obo`  
-     `git add imports/foo_import.owl`  
-     `git status` - only those two files should be added and ready to be committed. There will be some untracked files as well, which should not be added or committed.  
+1. Navigate to your local ontology directory, for example: `cd src/ontology`
+2. Create a git branch, e.g. `git checkout -b iss-GH_ISSUE_NUMBER`
+3. Run command: `sh run.sh make refresh-merged`
+4. Run `git status` to see what files have been updated
+5. Commit the updated files:
+     `git add <PATH-TO-FILE>`
+     `git status` - only the updated files should be added and ready to be committed. There will be some untracked files as well, which should not be added or committed.
      `git commit`  
      `git push`  
-8. Once you regenerated the new import, it could contain newly deprecatecd classes from the source ontology and this could affect the Mondo ontology by creating danglers/obsolete references. To fix this, follow the instructions in [Repair axioms pointing to deprecated classes](https://mondo.readthedocs.io/en/latest/developer-guide/repair-obsoleted-classes/).
+6. Once the new imports are generated, it could contain newly deprecatecd classes from the source ontology and this could affect the Mondo ontology by creating danglers/obsolete references. To fix this, follow the instructions in [Repair axioms pointing to deprecated classes](https://mondo.readthedocs.io/en/latest/developer-guide/repair-obsoleted-classes/).
```

Comment:
typo: "newly deprecatecd classes" should be "newly deprecated classes"

Reply:
thanks for the comment on the typo. i did not re-write the entire step and missed that.

Reply:
these are the instructions from Nico

Reply:
The git steps are the original ones for this page. As part of the Documentation clean-up and re-organization, I'd suggest there be a page with both GitHub Desktop and git command line that briefly goes over a few common actions (https://github.com/monarch-initiative/mondo/issues/7015) and be referenced in the pages as needed for common git commands.

Reply:
Also, for the future it's fine to hold off on Approval until you have time to run the command and verify.